src/transforms/without.ts

```diff
@@ -0,0 +1,41 @@
+import { ComparePredicate } from "../types";
+import { createFilterIterable } from "./filter";
+import { IterableCreatorIterable } from "../IterableCreatorIterable";
+
+export const createWithoutIterable = <TItem>(
+  iterator: Iterable<TItem>,
+  withoutItems: Iterable<TItem>,
+  comparePredicate: ComparePredicate<TItem> | void = void 0
```

Comment:
Could use the optional parameter syntax here as well:

```suggestion
  comparePredicate?: ComparePredicate<TItem>
```